File: PHP/TrianguloPoligonosRegularesTest.php
<?php
/**
 * @version PHP
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ ./vendor/bin/phpunit NomeDaClasseTest.php
 */
namespace PHP;

use PHPUnit\Framework\TestCase;
require ("TriangulosPoligonosRegulares.php");

class TriangulosPoligonosRegularesTest extends TestCase {

    public function testTriangulosPoligonosRegulares() {
        $obj = new TriangulosPoligonosRegulares();
        //Test 1
        $want = 1;
        $got = $obj->minTriangules(3);
        $this->assertEquals($want, $got);
        //Test 2
        $want = 2;
        $got = $obj->minTriangules(4);
        $this->assertEquals($want, $got);
        //Test 3
        $want = 3;
        $got = $obj->minTriangules(5);
        $this->assertEquals($want, $got);
    }

}

File: JAVA/TrianguloPoligonosRegularesTest.java
/** 
 * @version JAVA
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ mvn clean test -Dtest=your.package.TestClassName
 */ 
package JAVA;

import static org.junit.Assert.assertEquals;
import org.junit.Test;

public class TriangulosPoligonosRegularesTest {

	@Test
	public void testTriangulosPoligonosRegulares() throws Exception {
		//Test 1
		int want = 1;
		int got = TriangulosPoligonosRegulares.minTriangules(3);
		assertEquals(want, got);
		//Test 2
		want = 2;
		got = TriangulosPoligonosRegulares.minTriangules(4);
		assertEquals(want, got);
		//Test 3
		want = 3;
		got = TriangulosPoligonosRegulares.minTriangules(5);
		assertEquals(want, got);
	}

}
